docs/UserGuide.md

```diff
@@ -96,7 +98,9 @@ INDEX must be a positive integer.
 **Format:** `add NAME d/{SOCIAL MEDIA PLATFORM}, {ID}` or `add INDEX d/{SOCIAL MEDIA PLATFORM}, {ID}`
 
 NAME is case sensitive and must match the full name.</br>
-INDEX must be a positive integer.
+INDEX must be a positive integer. </br>
```

Comment:
not sure if all </br>'s are working now, since the original UG displayed properly on my machine. Can anyone with a Mac check?